src/components/Renderer/widgets/widget-util.tsx

```diff
@@ -154,7 +154,7 @@ export function getObjectPropertyNames({
 	}
 	const uiSchemaPropertyNames = getMaterializedPropertyNames(uiSchema);
 	const schemaPropertyNames =
-		get(uiSchema, ['ui:order'], keys(get(schema, 'properties'))) || [];
+		(uiSchema?.['ui:order'] ?? keys(schema.properties)) || [];
```

Comment:
This does mean that passing `null` now defaults to `keys(schema.properties)` instead of `[]` - I think that's correct but if not then let me know